server/pulp/server/async/tasks.py

```diff
@@ -456,6 +462,8 @@ def on_failure(self, exc, task_id, args, kwargs, einfo):
         else:
             _logger.info(_('Task failed : [%s]') % task_id)
             # celery will log the traceback
+        if 'scheduled_call_id' in kwargs:
```

Comment:
This seems right, but to recap the expected behavior, any task (including the ones like `queue_publish`) that has `schedule_call_id` passed in as a kwarg and fails will increment the failure count. That seems right to me.